.circleci/config.yml

```diff
@@ -151,7 +151,8 @@ jobs:
       - run:
           name: Publish Docker Image to ECR
           command: |
-            IMAGE_TAG=${CIRCLE_TAG/v/''}
+            IMAGE_TAG=${CIRCLE_TAG}
+            IMAGE_TAG=$(echo $IMAGE_TAG | cut -c 2-)
```

Comment:
Can these 2 lines be combined to `IMAGE_TAG=$(echo $CIRCLE_TAG | cut -c 2-)` ?

Reply:
I think they can be, I'll make that change.

Reply:
Made the change.